internal/manifest/list.go

```diff
@@ -82,17 +83,21 @@ type ListEdit struct {
 	ListOperation ListOp
 
 	// if Op == ListEditUpdate (basically the previous UpdateInstances). All fields must be set.
-	UpdateOldDigest digest.Digest
-	UpdateDigest    digest.Digest
-	UpdateSize      int64
-	UpdateMediaType string
+	UpdateOldDigest             digest.Digest
+	UpdateDigest                digest.Digest
+	UpdateSize                  int64
+	UpdateMediaType             string
+	OverwriteAnnotations        bool
```

Comment:
- Name this `Update…` to make it lear it relates to `ListEditUpdate`, please. [1]
- Per the other thread, the `UpdateInstances` caller will not set `UpdateCompressionAlgorithms`, so we must have an opt-in into both kinds of the annotation changes. Maybe `UpdateAffectAnnotations`? `UpdateDoNotChangeAnnotations`? (with opposite meanings).

---

[1] Or maybe
```go
type ListEditStruct {
  ListOperation …

  // if Op == ListEditUpdate …
  Update struct {
    OldDigest …
    Digest …
  }
```
I haven’t checked that is actually possible. It would mean a tiny bit less repetition here, at the cost of extra `.` in all users; I don’t have a preference.